waku/common/crypto.nim

```diff
@@ -0,0 +1,3 @@
+import ./utils/crypto
+
+export crypto
```

Comment:
Please, move the utils code here. We should reduce the code inside the `utils` module to the minimum.